src/KeycloakTypings.d.ts

```diff
@@ -0,0 +1,345 @@
+import * as express from 'express'
+
+/**
+ * The JavaScript module is exported as a single function, but for TypeScript we
+ * need to export the function and a set of interfaces so developers can assign
+ * types such as Grant, Token, etc. to variables in their own code.
+ * 
+ * To achieve this we export "KeycloakConnect" that references a namespace
+ * containing our typings, and a static instance exposing the constructor
+ */
+declare const KeycloakConnect: KeycloakConnectStatic
+export = KeycloakConnect
+
+interface KeycloakConnectStatic {
+  new (options: KeycloakConnect.KeycloakOptions, config: KeycloakConnect.KeycloakConfig): KeycloakConnect.Keycloak
+}
+
+declare namespace KeycloakConnect {
```

Comment:
This is amazing. How about we create extra PR to add some types to 
https://github.com/keycloak/keycloak-nodejs-connect/blob/master/package.json